pages/index.js

```diff
@@ -134,7 +134,7 @@ export default function Home() {
             textDecoration="underline"
             cursor="pointer"
           >
-            transaction
+            transaction: {txHash.hash}
```

Comment:
Shouldn't it be just `txHash` as when the function gets called its already passed as `tx.hash` ?

Reply:
@MarshBY I believe txHash is an object, and `txHash.hash` is the hash itself

Reply:
@personnamedmike `txHash` is the hash itself. Because if not, it'd be like doing `tx.hash.hash` . I just tried it on Rinkeby and this is what I got: 
![image](https://user-images.githubusercontent.com/57074511/188670116-b5fd5bfc-086c-4eb0-8347-e65b55b21bd1.png)

And on the console I got `txHash` is this:
![image](https://user-images.githubusercontent.com/57074511/188670319-d2c6d900-78cb-45d2-a326-0bcefd91b4f7.png)

Reply:
**transaction.hash ⇒ string< [DataHexString](https://docs.ethers.io/v5/api/utils/bytes/#DataHexString)< 32 > >** 
_____
https://github.com/rite-of-moloch/rom-frontend/blob/bb451fc7df855456bdb1e7ceb8b4eeade547dc47/pages/index.js#L155-L163